example/lib/main.dart

```diff
@@ -80,7 +80,18 @@ class _MyHomePageState extends State<MyHomePage> {
           style: TextStyle().copyWith(fontWeight: FontWeight.bold),
         ),
         const SizedBox(height: 12.0),
-        LinkText(text: _text, textAlign: TextAlign.center),
+        LinkText(
+          text: _text,
+          textAlign: TextAlign.center,
+          beforeLaunch: (uri) {
```

Comment:
btw, what's up with the onLinkTap --> beforeLaunch migration?

Reply:
soru meant in general with the two PRs, just wasn't sure where to stick the comment ^^"

Reply:
https://github.com/flutter/flutter/issues/106012